utilities/hco.py

```diff
@@ -236,6 +236,7 @@ def wait_for_hco_post_update_stable_state(admin_client, hco_namespace):
         namespace=hco_namespace.name,
     ):
         wait_for_dp(dp=deployment)
+        deployment.wait_for_replicas()
```

Comment:
isn't it the same action? to me it seems both `wait_for_dp` func and `wait_for_replicas` method duplicate each other